lib/owners/lexer.l

```diff
@@ -0,0 +1,21 @@
+class Parser
```

Comment:
This should somehow be namespaced in the `Owners` constant. That'll be tricky since it's a class. Might be worth changing to a module.